src/tests/ftest/container/boundary.py

```diff
@@ -1,145 +1,139 @@
-#!/usr/bin/python3
 """
   (C) Copyright 2022 Intel Corporation.
 
   SPDX-License-Identifier: BSD-2-Clause-Patent
 """
 
 import time
+import itertools
+import random
+
+from avocado.core.exceptions import TestFail
+
 from apricot import TestWithServers
 from general_utils import DaosTestError
-from avocado.core.exceptions import TestFail
 from thread_manager import ThreadManager
 
 
 class BoundaryTest(TestWithServers):
     """
-    Epic: Create system level tests that cover boundary tests and
-          functionality.
+    Epic: Create system level tests that cover boundary tests and functionality.
     Testcase:
           DAOS-8464: Test lots of pools and connections
     Test Class Description:
           Start DAOS servers, create pools and containers to the support limit.
     :avocado: recursive
     """
 
-    def create_container_and_test(self, pool=None, pool_num=1, cont_num=1, with_io=False):
+    def __init__(self, *args, **kwargs):
+        """Initialize a BoundaryTest object."""
+        super().__init__(*args, **kwargs)
+        self.with_io = False
+        self.io_run_time = None
+        self.io_rank = None
+        self.io_obj_classs = None
+
+    def setUp(self):
+        """Set Up BoundaryTest"""
+        super().setUp()
+        self.pool = []
+        self.with_io = self.params.get("with_io", '/run/boundary_test/*')
+        self.io_run_time = self.params.get("run_time", '/run/container/execute_io/*')
+        self.io_rank = self.params.get("rank", '/run/container/execute_io/*')
+        self.io_obj_classs = self.params.get("obj_classs", '/run/container/execute_io/*')
+
+    def create_pool(self):
+        """Get a test pool object and append to list.
+
+        Returns:
+            TestPool: the created test pool object.
+
+        """
+        pool = self.get_pool()
+        self.pool.append(pool)
+        return pool
+
+    def create_container_and_test(self, pool=None, cont_num=1):
         """To create single container on pool.
+
```

Comment:
While you're here, you might want to update line 60 to:
`pool (TestPool): ...`

Reply:
Thanks. I'll update if I need to push again